src/entity/ClaimableItem.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Column,
+  Entity,
+  Index,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import type { User } from './user';
+
+export enum ClaimableItemTypes {
+  Plus = 'plus',
+}
+
+@Entity()
+export class ClaimableItem {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ type: 'text' })
+  @Index('IDX_claimable_item_email')
+  email: string;
+
+  @Column({ default: () => 'now()' })
+  createdAt: Date;
+
+  @Column({ nullable: true })
+  claimedAt?: Date;
```

Comment:
We might need join index for email/claimedAt due to query for existing items. But not blocking at the start.

Reply:
non blocking